operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -825,14 +840,21 @@ private V1Probe createReadinessProbe(TuningParameters.PodTuning tuning) {
         .timeoutSeconds(getReadinessProbeTimeoutSeconds(tuning))
         .periodSeconds(getReadinessProbePeriodSeconds(tuning))
         .failureThreshold(FAILURE_THRESHOLD)
-        .httpGet(httpGetAction(READINESS_PATH, getDefaultPort()));
+        .httpGet(
+            httpGetAction(
+                READINESS_PATH,
+                getLocalAdminProtocolChannelPort(),
+                isLocalAdminProtocolChannelSecure()));
     return readinessProbe;
   }
 
   @SuppressWarnings("SameParameterValue")
-  private V1HTTPGetAction httpGetAction(String path, int port) {
+  private V1HTTPGetAction httpGetAction(String path, int port, boolean useHTTPS) {
     V1HTTPGetAction getAction = new V1HTTPGetAction();
     getAction.path(path).port(new IntOrString(port));
+    if (useHTTPS) {
+      getAction.scheme("HTTPS");
```

Comment:
Shouldn't this be lower case, "https"?

Reply:
The docs I found suggest valid values for scheme are "HTTP" and "HTTPS"
https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#examples
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/